code/modules/research/techweb/nodes/alien_nodes.dm

```diff
@@ -86,6 +86,7 @@ ABDUCTOR_SUBTYPE_UNLOCKS(/datum/techweb_node/alien_engi)
 		"surgery_brainwashing_mechanic",
 		"surgery_heal_combo_upgrade_femto",
 		"surgery_zombie",
+		"surgical_processor", //BUBBERS EDIT: Surgical Processor addition to med lathe
```

Comment:
```suggestion
```

You would add the design to the techweb node in modular_skyrat/master_files/code/modules/research/techweb/all_nodes.dm